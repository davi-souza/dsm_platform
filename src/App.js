import React from 'react';
import { Switch, Route } from 'react-router-dom';
import AppBar from '@material-ui/core/AppBar';
import Toolbar from '@material-ui/core/Toolbar';
import Typography from '@material-ui/core/Typography';
import Quotation from './views/Quotation';
import './App.scss';

const style = {
	appBar: {
		backgroundColor: 'transparent',
	},
};

function App() {
	return (
		<React.Fragment>
			<AppBar
				color="default"
				elevation={0}
				position="static"
				style={style.appBar}
			>
				<Toolbar>
					<Typography variant="h6" color="inherit">
						Mech4u
					</Typography>
				</Toolbar>
			</AppBar>
			<Switch>
				<Route component={Quotation} />
			</Switch>
		</React.Fragment>
	);
}

export default App;
