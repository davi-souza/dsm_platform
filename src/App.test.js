import React from 'react';
import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import App from './App';

configure({ adapter: new Adapter() });

describe('App tests', () => {
	it('should render without crash', () => {
		const component = shallow(<App />);
		expect(component).toMatchSnapshot();
	});
});
