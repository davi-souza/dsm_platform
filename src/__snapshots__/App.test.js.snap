// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App tests should render without crash 1`] = `ShallowWrapper {}`;
